src/protocols/uniswapOracle.js

```diff
@@ -0,0 +1,24 @@
+const { truffleContract } = require("../helpers");
+const uniswapOracleJson = require("../../build/contracts/UniswapOracle.json");
+
+module.exports = {
+  /**
+   * Deploys a Uniswap oracle.
+   * @param uniswapFactory the uniswap factory address
+   * @returns the Uniswap oracle as a truffleContract object
+   */
+  deploy: async (web3, owner, uniswapFactory) => {
+    const oracle = await truffleContract.new(
+      web3,
+      uniswapOracleJson.abi,
+      `0x${uniswapOracleJson.bytecode.replace(/0x/, "")}`,
```

Comment:
Does `bytecode` include multiple instances of `0x` ? If not, then you can just use `uniswapOracleJson.bytecode`

Reply:
That's fair, will change. It was copy pasted code to work with or without the 0x prefix. We don't need that here.